docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Future Release
         * Schema now maintains column order after renaming a column (:pr:`1594`)
     * Changes
         * Rename `backup_dtype` to `pyspark_dtype` (:pr:`1593`)
+        * Remove ``[0, 1]`` from boolean predictions (:pr:`1600`)
```

Comment:
Could we update this to say "Removed inference for ``["0", "1"]``, ``["0.0", "1.0"]``, and ``[0, 1]``" as ``Boolean`` logical types, but maintained forced inference of such values." or something like that?